src/MongoDB.Driver.Core/MongoDB.Driver.Core.csproj

```diff
@@ -129,12 +129,16 @@
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.5'">
     <PackageReference Include="System.Collections.Specialized" Version="4.0.1" />
     <PackageReference Include="System.Diagnostics.TextWriterTraceListener" Version="4.0.0" />
-    <PackageReference Include="System.Diagnostics.TraceSource" Version="4.0.0" />
+    <PackageReference Include="System.Diagnostics.TraceSource" Version="4.3.0" />
     <PackageReference Include="System.Net.NameResolution" Version="4.3.0" />
     <PackageReference Include="System.Net.Security" Version="4.3.2" />
     <PackageReference Include="System.Security.SecureString" Version="4.0.0" />
     <PackageReference Include="System.Threading.Thread" Version="4.3.0" />
   </ItemGroup>
+  
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
```

Comment:
I feel like some libraries from lines 129-137 should be also placed here. Should we create a ticket to investigate it (or maybe we already have such ticket, I don't remember)?

Reply:
I don't think we have a ticket to investigate this: feel free to create one so we can discuss it in triage :)

Reply:
How are we compiling without those lines then? Indirect dependencies?

Reply:
Maybe when targetting `netstandard2.0` some of those dependencies are automatic?